packages/ckeditor5-utils/src/emittermixin.ts

```diff
@@ -335,6 +338,64 @@ export default EmitterMixin;
  *
  * @interface Emitter
  */
+export interface Emitter {
+	on(
+		event: string,
+		callback: ( this: this, ev: EventInfo, ...args: any[] ) => void,
+		options?: CallbackOptions
+	): void;
+
+	once(
+		event: string,
+		callback: ( this: this, ev: EventInfo, ...args: any[] ) => void,
+		options?: CallbackOptions
+	): void;
+
+	off( event: string, callback: Function ): void;
+
+	listenTo(
+		emitter: Emitter,
+		event: string,
+		callback: ( this: this, ev: EventInfo, ...args: any[] ) => void,
+		options?: CallbackOptions
+	): void;
+
+	stopListening( emitter?: Emitter, event?: string, callback?: Function ): void;
```

Comment:
It won't work for instance when `stopListening(undefined, 'myevent')` Write overloads, like for `stopDelegating` to ensure all previous parameters are passed.